pom.xml

```diff
@@ -164,6 +164,12 @@
             <artifactId>httpclient</artifactId>
             <version>4.5.3</version>
         </dependency>
+        <dependency>
+            <groupId>net.sf.json-lib</groupId>
```

Comment:
Maybe we shall use Jackson instead of json-lib like in allure2?
https://github.com/allure-framework/allure2/blob/master/build.gradle#L102